src/components/LiquidityPoolInteraction.tsx

```diff
@@ -122,62 +67,62 @@ export const LiquidityPoolInteraction = ({
 	};
 
 	return (
-		<div className="p-6 md:p-12 md:pr-36 bg-neutral-0 space-y-7">
-			<h2 className="space-x-5 flex items-center">
-				<Icon name="line-chart" className="w-7 h-7" />
-				<span className="uppercase font-bold text-2xl">{action}</span>
-			</h2>
-			<InvestmentDetails
-				balance={userBaseBalance}
-				balanceCurrency="USDC"
-				investments={userStake}
-				investmentsCurrency="USDC"
+		<Form
+			name="invest"
+			form={form}
+			onFinish={onFinish}
+			onFinishFailed={onSubmitFailed}
+			layout="vertical"
+		>
+			<Input
+				name="amount"
+				label={intl.formatMessage(MESSAGES.amountInputLabel)}
+				step="0.1"
+				className="bg-credix-primary"
+				placeholder={intl.formatMessage(MESSAGES.amountInputPlaceholder)}
+				labelClassName="mb-4"
+				lang="en"
+				type="number"
+				required={true}
+				rules={[
+					{ required: true },
```

Comment:
does this need a message?

Reply:
It technically doesn't as it has a default message. But I've added one for better i18n support.